third_party/llvm-project/dwarf2yaml.cpp

```diff
@@ -120,6 +120,7 @@ void dumpDebugLoc(DWARFContext &DCtx, DWARFYAML::Data &Y) { // XXX BINARYEN
   uint64_t offset = 0;
   DWARFDebugLoc locList;
   while (locsData.isValidOffset(offset)) {
+    uint64_t locListOffset = offset;
```

Comment:
please add an `// XXX BINARYEN` comment on lines added in third party, like here (see line 120 for example). it makes it easier to see what we've modified locally.

Reply:
done